pyproject.toml

```diff
@@ -7,13 +7,14 @@ requires-python = ">=3.11"
 dependencies = [
     "asgiref>=3.8.1",
     "celery[redis]>=5.5.2",
-    "daphne>=4.1.2",
-    "django>=5.2",
+    "daphne==4.2.1",
+    "django==5.2.7",
```

Comment:
Out of curiosity, why did you pin the version of `daphne` and `django`, when most other packages just set the minimal required version?